### ISSUE TITLE
Silent requires device to be set

### ISSUE THREAD

**Issue by FreekingDean** [rancher/k3os#205]
When installing using the kernel arguemnt `k30s.install.silent=true` I was running into where it would display the help text of the install script. I realized it was not setting a device, so I used a kernel argument to explicitly set the device which worked fine.

If I remove silent, the install script intuits the correct device automatically.

**Comment by dweomer** [comment]
The documentation could be better for silent installations.

**Comment by FreekingDean** [comment]
Yeah, could totally just be a documentation solve! Happy to open that PR if thats the route you'd like to go!

**Comment by zenekron** [comment]
Hit this today, had a rough half an hour.
At least for the time being it'd be nice to have a note in `k3os.install.silent`'s description documenting this kind of behaviour..
